applications/serial_lte_modem/src/gnss/Kconfig

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+config SLM_GNSS
```

Comment:
It looks like currently it's not possible to enable only cellular positioning, i.e. GNSS is always enabled at the same time. Not sure if this is a problem, though. The highest level configuration could be "positioning", with options for "GNSS" and "cell pos" under that. And "A-GPS" and "P-GPS" under "GNSS".

Reply:
Yes, but AT#XCELLPOS will never start GPS on modem side